examples/calculator_qml/main.qml

```diff
@@ -8,60 +8,14 @@ Window {
     height: 1000
     visible: true
 
-    ChartView {
```

Comment:
Replace ChartView by Button. This component is only to avoid having WidgetView at position 0, 0.